### DISCUSSION TITLE
Next/Image for remote images

### DISCUSSION THREAD

**Discussion by jakubhome** [vercel/next.js#71056] [Help]
### Summary

If I use Next/Image for product images (e.g. ecommerce product listing) that come from my hosting (remote source) and are dynamic, should I still take full advantage of Next/Image optimization, or should I set unoptimized: true in next config?

I care as much as possible about Web Vitals pointers, and my hosting does not offer good compression like cdn.

**Comment by chloegng** [comment]
If you care about Web Vitals and the performance of your site, you should take full advantage of Next/Image optimization, even if your images are coming from a remote source. Using Next/Image provides benefits such as automatic image resizing, lazy loading, and format conversion, which can significantly enhance the loading experience.

Setting `unoptimized: true` in your Next.js config would bypass these optimizations, potentially leading to slower load times and poorer Web Vitals scores. Instead, consider the following tips:

1. **Use a CDN**: If possible, host your images on a CDN that offers better compression and faster delivery. This will complement Next/Image's optimization features.

2. **Optimize Images on Upload**: Ensure that images are optimized before they are uploaded to your hosting. This can include resizing, compressing, and converting to modern formats like WebP.

3. **Image Placeholders**: Use low-quality image placeholders to improve perceived loading performance.

4. **Proper Dimensions**: Always specify width and height for your images to avoid layout shifts.

By leveraging Next/Image alongside a better image hosting strategy, you can achieve optimal performance and improve your site's Web Vitals scores.
